test/test-create-options.js

```diff
@@ -12,13 +12,16 @@ test('create-options:', function (t) {
     module: {envVar: {testenvVar: 'thisisatest'}}
   };
 
+  //ensure that there is no environment variable defined
+  delete process.env['npm_config_tmp'];
```

Comment:
setting to null doesn't seem to work, npm by default sets `npm_config_tmp` when running `npm test` so we have to unset this to check that citgm can run without that environment variabale set